packages/stepper/src/components/Step.js

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+import Container from "../styledComponents/Step";
+import statusTypes from "../constants/statusTypes";
+
+export default class Step extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick(event) {
+    event.stopPropagation();
+
+    const isCurrentStep = this.props.status === statusTypes.CURRENT;
+
+    if (!isCurrentStep) {
+      return;
+    }
+
+    return this.props.onClick({
+      index: this.props.index,
+      value: this.props.value
+    });
+  }
+
+  render() {
+    const status = this.props.status;
+    const isCompleted = status === statusTypes.COMPLETED;
+
+    return (
+      <Container status={status} onClick={this.onClick}>
+        {this._renderIcon(isCompleted ? "wg-check" : "wg-checker")}
+
+        <div className="description">{this.props.value}</div>
```

Comment:
what do you think about replacing the ``this.props.value`` property with ``this.props.children``? It might be useful in the future. (it would allow for "steps" arrays that can be nodes later, like to allow rich text)